lib/helper/WebDriverIO.js

```diff
@@ -913,9 +963,74 @@ class WebDriverIO extends Helper {
    * ```
    */
   seeNumberOfVisibleElements(locator, num) {
+    return this.grabNumberOfVisibleElements.then(function (res) {
+      return assert.equal(res, num,
+          `expected number of visible elements (${locator}) is ${num}, but found ${res}`);
+    });
+  }
+
+  /**
+   * Checks that there is an Element with given CSS property.
+   *
+   * ```js
+   * I.seeCssPropertyOnElement('h3', 'font-weight', "bold");
+   * ```
+   */
+  seeCssPropertyOnElement(locator, cssProperty, value) {
```

Comment:
I still think this should accept cssProperties and values as object.
If user needs o check several CSS properties he will need to call `seeCssPropertyOnElement` several times, which 
will do extra job locating an element. 

Also, I don't think that we should check all possible elements on a page for having CSS property.
If I search for `h3 font-weight:bold` I want to ensure that h3 headings on page are bold. But I can also locate `h3.bold` element which adds boldness to text. Then it makes my test false-positive. I thought my h3s are bold but they are not! 

So this should go to the first element only!
So users would prepare more exact locators.